lib/delayed/railtie.rb

```diff
@@ -1,5 +1,6 @@
 require 'delayed_job'
 require 'rails'
+require 'active_job/queue_adapters/delayed_job_adapter'
```

Comment:
This should probably be in `lib/delayed_job.rb` using a similar `ActiveSupport.on_load` pattern as action mailer.